packages/pcd/zk-eddsa-frog-pcd/src/ZKEdDSAFrogPCD.ts

```diff
@@ -211,11 +207,16 @@ async function checkProveInputs(args: ZKEdDSAFrogPCDArgs): Promise<{
 function snarkInputForProof(
   frogPCD: EdDSAFrogPCD,
   identityPCD: SemaphoreIdentityPCD,
-  externalNullifer: string,
+  externalNullifier: string,
   watermark: string
 ): Record<string, `${number}` | `${number}`[]> {
   const frogAsBigIntArray = frogDataToBigInts(frogPCD.claim.data);
   const signerPubKey = frogPCD.proof.eddsaPCD.claim.publicKey;
+
+  // Note: unpackSignature leaves the R8 point's coordinates in Montgomery
+  // form, which is then reversed by toObject below.
+  // This is a reference to Montgomery form of numbers for modular
+  // multiplication, NOT Montgomery form of eliptic curves.  See https://en.wikipedia.org/wiki/Montgomery_modular_multiplication#Montgomery_form
```

Comment:
this comment duplication makes me wonder if we should extract conversion functionality into shared util files which could host this documentation so that it doesn't have to be duplicated

Reply:
Probably.  This will likely happen when I'm sure what libraries we want to use between POD and EdDSATickets.  I'm thinking of putting the final decision to that in a @pcd/crypto package or something like that.  These comments are an interim solution to get the info in the places where it needs to be noticed.